### DISCUSSION TITLE
Misterious syntax error

### DISCUSSION THREAD

**Discussion by Vadiml1024** [community/community#27898] [Actions] [answered]
Please help with this syntax error in job description
[https://github.com/Vadiml1024/python-libarchive/actions/runs/2775436659/workflow](url)

**Comment by airtower-luna** [comment, marked as answer]
Looks like the error message is misleading, the error isn't *on* line 10, it's in the job that starts at that line. The problem is here, and in all the lines below:

```yaml
    steps:
      - uses: actions/checkout@v2

    - name: Print System Information
```

`steps` must be a list, and in YAML lists all list items (marked by `-`) must be indented at the same level. So this would be correct:

```yaml
    steps:
      - uses: actions/checkout@v2

      - name: Print System Information
        run: |
          echo "uname -a: $( uname -a )"
          cat /etc/issue
          echo "Shell: $SHELL"
          echo "Mount points:"; mount
          env
```

Likewise for all the steps below.
